liquibase_changelog/patternatlas_componentHostingPatterns.xml

```diff
@@ -0,0 +1,174 @@
+            <column name="name" value="Forces"/>
+            <column name="position" value="3"/>
+            <column name="type" value="any"/>
+            <column name="pattern_schema_pattern_language_id" value="ddbd6d47-6a71-4e41-8704-8c313f3e819f"/>
+        </insert>
+        <insert tableName="pattern_section_schema">
+            <column name="id" value="82d8e696-a828-4346-a236-cced4542bc08"/>
+            <column name="label" value="Solution"/>
+            <column name="name" value="Solution"/>
+            <column name="position" value="4"/>
+            <column name="type" value="any"/>
+            <column name="pattern_schema_pattern_language_id" value="ddbd6d47-6a71-4e41-8704-8c313f3e819f"/>
+        </insert>
+        <insert tableName="pattern_section_schema">
+            <column name="id" value="12d8e696-aef8-1246-ae32-405687d5a2ce"/>
+            <column name="label" value="License"/>
+            <column name="name" value="License"/>
+            <column name="position" value="5"/>
+            <column name="type" value="any"/>
+            <column name="pattern_schema_pattern_language_id" value="ddbd6d47-6a71-4e41-8704-8c313f3e819f"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="lharzenetter" id="1674138915768-4">
+        <insert tableName="pattern">
+            <column name="id" value="cdbd8bf8-84ac-49f6-a58e-e7414333947c"/>
+            <column name="name" value="Customizable Deployment Stack Pattern"/>
+            <column name="uri" value="https://patternpedia.org/patternLanguages/componentHostingPatterns/customizableDeploymentStackPattern"/>
+            <column name="content" value="{&quot;Icon&quot;: &quot;Enter your input for this section here.&quot;, &quot;Forces&quot;: &quot;Enter your input for this section here.&quot;, &quot;Context&quot;: &quot;Enter your input for this section here.&quot;, &quot;License&quot;: &quot;This pattern was first published in *Service-Oriented Computing. SummerSOC 2022. Communications in Computer and Information Science*, vol 1603, pp. 45-67, 2022 by Springer Nature.\n\nYussupov, V., Breitenbücher, U., Brogi, A., Harzenetter, L., Leymann, F., Soldani, J., **\&quot;Serverless or Serverful? A Pattern-Based Approach for Exploring Hosting Alternatives\&quot;**. In: Service-Oriented Computing. SummerSOC 2022. Communications in Computer and Information Science, vol 1603, pages 45-67. Springer, Cham. 2022, DOI: [10.1007/978-3-031-18304-1_3](https://link.springer.com/chapter/10.1007/978-3-031-18304-1_3)&quot;, &quot;Problem&quot;: &quot;How can a software component be hosted if it requires customization of the underlying infrastructure or host environment on which it runs?&quot;, &quot;Solution&quot;: &quot;Enter your input for this section here.&quot;}"/>
+            <column name="icon_url" value="https://raw.githubusercontent.com/PatternAtlas/pattern-atlas-content/addComponentHostingPatterns/icons/Component_Hosting_Patterns/customizable-stack.png"/>
```

Comment:
I don't know what happens if anyone ever deletes the addComponentHostingPatterns branch and the pattern logos still reference this. Are you sure this will still be reachable?
Otherwise a permalink to the file could work: https://docs.github.com/de/repositories/working-with-files/using-files/getting-permanent-links-to-files